drivers/flash/nrf_qspi_nor.c

```diff
@@ -661,6 +661,10 @@ static int qspi_nrfx_configure(const struct device *dev)
 	nrfx_err_t res = nrfx_qspi_init(&QSPIconfig, qspi_handler, dev_data);
 	int ret = qspi_get_zephyr_ret_code(res);
 
+	if (ret == 0 && !nrf52_errata_121()) {
+		nrf_qspi_iftiming_set(NRF_QSPI, DT_INST_PROP(0, rx_delay));
```

Comment:
Perhaps it would be better to use `#if DT_INST_NODE_HAS_PROP(0, rx_delay)` to modify this setting only when the property is actually set in devicetree (and remove this `default: 2` in the property definition).

Reply:
This did occur to me, and I agree.